README.md

```diff
@@ -46,16 +46,22 @@ If you're not familiar with git, consider learning the basics from one of the ma
 
 In order to start contributing, simply follow these 10 steps:
 
-1. Please make sure you've read the [CONTRIBUTING.md](https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md) file.
+1. Please make sure you've read through the [CONTRIBUTING.md](https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md) file.
 2. Fork the [ACS UPB Mobile](https://github.com/acs-upb-mobile/acs-upb-mobile) repository.
-3. Clone the fork onto your system and open it as an Android Studio project.
+3. Clone the fork onto your system (`git clone`) and open it as an Android Studio project.
 4. Don't forget to run `flutter pub get` to update the dependencies.
+
+|:question:|Something isn't working, I see a bunch of errors! What do I do?|
+|:--------:|:--------------------------------------------------------------|
+|  :bulb:  |Did you skip that part in the first paragraph about following the [Prerequisites](https://github.com/student-hub/acs-upb-mobile#prerequisites) section before moving forward? Make sure you have everything installed, and **the correct Flutter version checked out**. Finally, you can try the troubleshooting steps [here](https://github.com/student-hub/acs-upb-mobile/blob/master/README.md#not-working).|
+
+
```

Comment:
this section is gold for newbies:))) I do not see anything else to be added here since those sections provide each significant details.